### ISSUE TITLE
parse_label_id

### ISSUE THREAD

**Issue by mengpingzeng** [xinyu1205/recognize-anything#128]
I have an object detection dataset that only contains bbox and class annotations. how to get parse_label_id

**Comment by xinyu1205** [comment]
Hi, you can only finetuning the image tagging branch of RAM/RAM++, only with image class annotations.

**Comment by mengpingzeng** [comment]
Thank you for your reply

**Comment by mengpingzeng** [comment]
finetuning the image tagging branch of RAM++ is similar to multi-label classification training? how to make the  parse_label_id with  object detection dataset.

**Comment by xinyu1205** [comment]
1. yes.
2. image tagging branch only need image tags annotations.

**Comment by mengpingzeng** [comment]
I know that. but I want to pretrain, how to make the label
